src/account/SecuritySubmenu.tsx

```diff
@@ -0,0 +1,532 @@
+              <Text>Toggle verification done</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={handleResetAppOpenedState}>
+              <Text>Reset app opened state</Text>
+            </TouchableOpacity>
+          </View>
+
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={handleToggleBackupState}>
+              <Text>Toggle backup state</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={showDebugScreen}>
+              <Text>Show Debug Screen</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={Sentry.nativeCrash}>
+              <Text>Trigger a crash</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={wipeReduxStore}>
+              <Text>Wipe Redux Store</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={confirmAccountRemoval}>
+              <Text>App Quick Reset</Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+      )
+    }
+  }
+
+  const getKeylessBackupItem = () => {
+    if (!showKeylessBackup) {
+      return null
+    }
+    if (deleteKeylessBackupStatus === KeylessBackupDeleteStatus.InProgress) {
+      return (
+        <SettingsItemCta
+          title={t('keylessBackupSettingsTitle')}
+          onPress={() => {
+            // do nothing
+          }}
+          testID="KeylessBackup"
+          cta={
+            <>
+              <LoadingSpinner width={22} />
+              <Text testID={`KeylessBackup/cta`} style={styles.value}>
+                {t('pleaseWait')}
+              </Text>
+            </>
+          }
+        />
+      )
+    } else if (cloudBackupCompleted) {
+      return (
+        <SettingsItemCta
+          title={t('keylessBackupSettingsTitle')}
+          onPress={onPressDeleteKeylessBackup}
+          testID="KeylessBackup"
+          showChevron
+          cta={
+            <>
+              <Text testID={`KeylessBackup/cta`} style={styles.value}>
+                {t('delete')}
+              </Text>
+            </>
+          }
+        />
+      )
+    } else {
+      return (
+        <SettingsItemCta
+          title={t('keylessBackupSettingsTitle')}
+          onPress={onPressSetUpKeylessBackup}
+          testID="KeylessBackup"
+          showChevron
+          cta={
+            <>
+              <Text testID={`KeylessBackup/cta`} style={styles.value}>
+                {t('setup')}
+              </Text>
+            </>
+          }
+        />
+      )
+    }
+  }
+
+  return (
+    <SafeAreaView>
```

Comment:
there's some regression here that i can't exactly point to but the bottom content is colliding into the bottom bar - can we apply the usual pattern of adding a `edges={['top']}` here and adding a padding bottom to the scroll view equal to the insets.bottom?

![Simulator Screenshot - iPhone 15 Pro - 2024-08-28 at 14 54 55](https://github.com/user-attachments/assets/a5a12e21-94ab-4bb9-bdae-54e619ef8e0f)

Reply:
it's also affecting the bottom sheets
![Simulator Screenshot - iPhone 15 Pro - 2024-08-28 at 14 56 16](https://github.com/user-attachments/assets/810da8de-8223-40a6-9321-01a32535224a)

Reply:
Ahh yes! Thank you for catching this. I've gone back and modified every new screen to use the correct pattern and verified that bottom sheets/scrolling etc. works as intended.